src/ModelContextProtocol.Core/Protocol/CompletionsCapability.cs

```diff
@@ -23,15 +20,4 @@ namespace ModelContextProtocol.Protocol;
 public sealed class CompletionsCapability
```

Comment:
Why do we need to keep this empty class?

Reply:
Ah, I now notice the remark about it being empty in the spec. Could you check that this is still the case with the latest revision and whether keeping this around is still necessary?